test/UnitTests/MSTest.Analyzers.UnitTests/PreferTestCleanupOverDisposeAnalyzerTests.cs

```diff
@@ -17,15 +17,30 @@ public async Task WhenTestClassHasDispose_Diagnostic()
             using Microsoft.VisualStudio.TestTools.UnitTesting;
 
             [TestClass]
-            public class MyTestClass : IDisposable
+            public class MyTestClass : IDisposable, IMyInterface
             {
                 public void [|Dispose|]()
                 {
                 }
             }
+            public interface IMyInterface { }
             """;
+        string fixedCode = """
+            using System;
+            using Microsoft.VisualStudio.TestTools.UnitTesting;
 
-        await VerifyCS.VerifyAnalyzerAsync(code);
+            [TestClass]
+            public class MyTestClass : IMyInterface
```

Comment:
what if it's inheriting a base class that implements `IDisposable`?

Reply:
I can't delete it in that case as it could have more code and it should be removed manually from the code